projects/nucleaizer.md

```diff
@@ -0,0 +1,63 @@
+---
+title: NucleAIzer
+layout: page
+---
+
+- *Contributor(s)*: Ervin Tasnádi (@etasnadi), Peter Horvath
+- *References*: [Project Site](nucleaizer.org), [Github](https://github.com/spreka/biomagdsb), [Publication](https://www.sciencedirect.com/science/article/pii/S2405471220301174)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
+
+### Why does this matter to biologists?
+
+### Why does this matter to you and your professional goals and responsibilities?
+
+## Goals
+> What would you like to accomplish during the Alfa Cohort collaboration?
```

Comment:
- Use Napari to detect the objects in the opened image using the NucleAIzer model.
- Add the functionality to Napari to train a new nucleAIzer model on your own dataset using transfer learning.